lib/Targets/AIETargets.cpp

```diff
@@ -31,6 +31,10 @@
 #include "llvm/Support/FileSystem.h"
 #include "llvm/Support/JSON.h"
 
+extern "C" {
+#include "cdo_driver.h"
```

Comment:
@stephenneuendorffer this is the spot you missed I believe (see warnings below in file view...)